pages/chat/[[...roomId]].js

```diff
@@ -169,15 +225,25 @@ export default function RoomId() {
                 <br />
                 <details open>
                     <summary><h3 style={{ display: 'inline-block', marginBottom: '1rem' }}>{ t('Rooms') }</h3></summary>
-                    { otherRooms && otherRooms.map((room) => <SidebarListEntry key={room.roomId} room={room} />) }
+                    { otherRooms && otherRooms.map((room) => {
+                        return <ServiceTable><SidebarListEntry key={room.roomId} room={room} /></ServiceTable>;
+                    }) }
                 </details>
                 <br />
             </IframeLayout.Sidebar>
-            { roomId && (
-                <IframeLayout.IframeWrapper>
-                    <iframe src={`${getConfig().publicRuntimeConfig.chat.pathToElement}/#/room/${roomId}`} ref={iframe} />
-                </IframeLayout.IframeWrapper>
-            ) }
+
+            { (deviceType !== 'mobile' || (deviceType === 'mobile' && roomId) || (deviceType === 'mobile' && !isRoomListVisible)) && <IframeLayout.IframeWrapper>
+                <IframeLayout.IframeHeader>
```

Comment:
I think we don't want the `<IframeLayout.IframeHeader>` component in /chat anymore... it looks like this now:
![Screenshot 2023-08-16 at 15-11-42 _spaces](https://github.com/medienhaus/medienhaus-spaces/assets/706419/a02d77e9-9a4c-4e9a-929a-a5bca807ca44)

... with two headers. When one is just enough.

Maybe check out what /chat looks like on `main` once more...